src/course-unit/data/slice.js

```diff
@@ -7,6 +7,9 @@ const slice = createSlice({
   name: 'courseUnit',
   initialState: {
     savingStatus: '',
+    isQueryPending: false,
+    newUnitId: null,
+    isTitleEditFormOpen: false,
```

Comment:
```suggestion
    isEditTitleFormOpen: false,
```
Can you change words order: `Edit` first and then `Title`. Seems like it will be more readable. The same for `changeTitleEditFormOpen` -> `changeEditTitleFormOpen`

Reply:
Renamed